plasmapy/diagnostics/charged_particle_radiography.py

```diff
@@ -1115,118 +1260,144 @@ def max_deflection(self):
 
         return max_deflection * u.rad
 
-    # *************************************************************************
-    # Synthetic diagnostic methods (creating output)
-    # *************************************************************************
-
-    def synthetic_radiograph(
-        self, size=None, bins=[200, 200], ignore_grid=False, optical_density=False
-    ):
+    def _enforce_order(self):
         r"""
-        Calculate a "synthetic radiograph" (particle count histogram in the
-        image plane).
 
-        Parameters
-        ----------
-        size : `~astropy.units.Quantity`, shape (2,2)
-            The size of the detector array, specified as the minimum
-            and maximum values included in both the horizontal and vertical
-            directions in the detector plane coordinates. Shape is
-            [[hmin,hmax], [vmin, vmax]]. Units must be convertable to meters.
+        The Tracker methods could give strange results if setup methods
```

Comment:
```suggestion
        The `Tracker` methods could give strange results if setup methods
```

There is also a blank line above this line that needs deleting.